x-pack/plugins/maps/public/layers/sources/es_search_source/scaling_form.tsx

```diff
@@ -68,8 +66,10 @@ export class ScalingForm extends Component<Props, State> {
 
   async loadIndexSettings() {
     try {
-      const indexPattern = await indexPatternService.get(this.props.indexPatternId);
-      const { maxInnerResultWindow, maxResultWindow } = await loadIndexSettings(indexPattern.title);
+      const indexPattern = await getIndexPatternService().get(this.props.indexPatternId);
+      const { maxInnerResultWindow, maxResultWindow } = await loadIndexSettings(
+        indexPattern!.title
```

Comment:
`!.` ?

Reply:
`!.` is typescript assert not null operator. Typescript thinks `indexPatternService.get` could return undefined for some reason. `indexPatternService.get` throws when the index pattern can not be found so indexPattern is always returned and can never be undefined.

Reply:
"Observability layers" doesn't really fit with the other descriptions, which typically include the word "data".  Maybe you can iterate on this suggestion:

APM real-time user monitoring data
APM Real User Monitoring data

and then make it more generic when you add more Observability data.

For Choropleth, I like @thomasneirynck's suggestion: World boundaries 

I'm not sure how well RUM and SLA are known.  I suggest spelling out:

Transaction duration (currently, there's a typo)
Percentage of Service Level Agreements 

APM Real User Monitoring Performance
APM Real User Monitoring Traffic

If you don't want to spell out in the menu, maybe spelling it out in the data source description is good enough.